modules/release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -194,7 +194,37 @@ With the release of the {product-short} 1.2.1 Helm chart, this is fixed.
 //link:{LinkRHIDPIssue}RHIDP-2931[RHIDP-2931]
 --
 
+== Fixed issues in {product} 1.2.2
 
+This section lists fixed issues with {product} 1.2.2:
+
+Dynamic plugins fail to load if the file name in ConfigMap is invalid::
++
+--
+Previously, an invalid dynamic plugin file name in ConfigMap prevented the creation of a Backstage CR. With this release, the Backstage CR can be created even with an invalid plugin file name, and a corresponding log message will be displayed if the RHDH build fails.
```

Comment:
For consistency...in this section anyway:

```suggestion
Previously, an invalid dynamic plugin file name in ConfigMap prevented the creation of a Backstage CR. With this release, the Backstage CR can be created even with an invalid plugin file name, and a corresponding log message will be displayed if the {product-short} build fails.
```

Reply:
@hmanwani-rh I also noticed an invalid attribute in _1.1. Upgrading the Red Hat Developer Hub Helm Chart_

> Log in to the OpenShift cluster as the cluster administrator and switch to the project or namespace in which {prod-short} was installed.

Do you want to fix it in this release update?

Reply:
Sure, fixed it